G1ANT.Addon.MSOffice/G1ANT.Addon.MSOffice/Structures/OutlookMailStructure.cs

```diff
@@ -140,5 +140,20 @@ protected override MailItem Parse(string value, string format = null)
         {
             return null;
         }
+
+        private string GetRecipientListOfType(OlMailRecipientType recipientType)
+        {
+            var recipients = string.Empty;
+
+            foreach (Recipient recipient in Value.Recipients)
+            {
+                if (recipient.Type == (int)recipientType)
+                {
+                    recipients += recipient.Address + ";";
+                }
+            }
+
+            return recipients;
```

Comment:
whole function can be written as one liner.

`            return Value.Recipients.Cast<Recipient>().Where(recipient => recipient.Type == (int)recipientType).Aggregate(string.Empty, (current, recipient) => current + (recipient.Address + ";"));
`